addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -5217,9 +5217,9 @@ X[]
                             contentBefore: '<p>a[bc</p><table><tbody><tr><td>a]b</td><td>cd</td><td>ef</td></tr></tbody></table>',
                             contentAfterEdit: '<p>a[bc</p>' +
                                             '<table class="o_selected_table"><tbody><tr>' +
-                                                '<td class="o_selected_td">a]b</td>' +
+                                                '<td class="o_selected_td">ab</td>' +
                                                 '<td class="o_selected_td">cd</td>' +
-                                                '<td class="o_selected_td">ef</td>' +
+                                                '<td class="o_selected_td">]ef</td>' +
```

Comment:
If table will be completely selected why not `ef]` rather than `]ef`?
Same question for all the test cases below?